src/Shared/AppModels/AppRoot.cs

```diff
@@ -90,6 +94,11 @@ public async Task InitAsync(CancellationToken cancellationToken = default)
 
             cancellationToken.ThrowIfCancellationRequested();
 
+            if (_appDebug is null)
+            {
+                _appDebug = new AppDebug();
```

Comment:
No need to assign this in `InitAsync`. We want debug functionality to work right away when `AppRoot` is constructed, so let's assign it from the field or the constructor, and remove nullability.